lightning-transaction-sync/src/lib.rs

```diff
@@ -68,6 +68,9 @@ extern crate bdk_macros;
 #[cfg(any(feature = "esplora-blocking", feature = "esplora-async"))]
 mod esplora;
 
+#[cfg(any(feature = "esplora-blocking", feature = "esplora-async"))]
+mod types;
```

Comment:
`types` is a bit too generic a name, IMHO. Would be better to have a specific -- yet short -- name lest it become a free-for-all to add anything. Would probably lean towards `state`.

Reply:
Sure, I'd prefer `common` over `types`.